UnityProject/Assets/Scripts/Tool/SpaceCleaner.cs

```diff
@@ -13,7 +13,7 @@ public class SpaceCleaner : NetworkBehaviour, ICheckedInteractable<AimApply>
 
 	[SerializeField]
 	[Range(1,50)]
-	private int reagentsPerUse = 5;
```

Comment:
why Change to 1?

Reply:
The change to 1 is because it's the amount of reagent now used per-tile since that was easier to implement.

Reply:
cool, Can it handle decimals?

Reply:
Yep, it can handle any kind of number.